rbc/typesystem.py

```diff
@@ -1118,7 +1121,10 @@ class Boolean1Model(datamodel.models.BooleanModel):
         def get_data_type(self):
             return self._bit_type
 
-    class Boolean8(nb.types.Boolean):
+    class Boolean8(nb.types.Integer):
+
+        def __init__(self, name):
+            super().__init__(name, bitwidth=8, signed=True)
```

Comment:
By inheriting from Integer rather than Boolean, one can get all `int8` overloads for free. Which means, no more errors like the one below:
https://github.com/xnd-project/rbc/blob/6fa215ddbaee50f1a16dd47ed0481ed411631a70/rbc/tests/test_omnisci_math.py#L180-L183